Documentation/Storage-Configuration/Advanced/ceph-configuration.md

```diff
@@ -23,6 +23,12 @@ to also change `ROOK_OPERATOR_NAMESPACE` to create a new Rook Operator for each
 forget to set `ROOK_CURRENT_NAMESPACE_ONLY`), or you can leave it at the same value for every
 Ceph cluster if you only wish to have one Operator manage all Ceph clusters.
 
+!!! note
+    The namespace `ROOK_OPERATOR_NAMESPACE` must be created manually if it does not exist:
```

Comment:
If they are configuring a different namespace, but the operator and cluster namespace are the same, this is not needed. How about this? And to keep the note simple, IMO we don't need to give them the example command to create the namespace.
```suggestion
    If the operator namespace is different from the cluster namespace, the operator namespace must be created before running the steps below. (The cluster namespace does not need to be created first, as it will be created by `common.yaml` in the script below.)
```